CHANGELOG.develop

```diff
@@ -1054,6 +1087,7 @@ Benner and Paweł Siudak):
 - Fixed "Invalid face reference" error in Helm transient state (thanks to
   duianto)
 - Hide lighter (thanks to Sylvain Benner)
+- Move helm variable from helm layer to spacemacs-completion layer (thanks to Kalle Lindqvist)
```

Comment:
This can be omitted because there is already an entry that covers the change (look for "Renamed helm variables and made them layer variables").  I added that entry earlier because there were a lot of related changes, and once I had sorted those changes out, I just added a single entry for all of them.  Sorry for the confusion!